core/new-gui/src/app/workspace/types/workflow-common.interface.ts

```diff
@@ -26,6 +26,7 @@ export interface OperatorPredicate
     outputPorts: { portID: string; displayName?: string }[];
     showAdvanced: boolean;
     isDisabled?: boolean;
+    isCached?: boolean;
```

Comment:
I believe this is backward compatible with the existing operators saved in db, just want to confirm.

Reply:
yes